trappy/plotter/ILinePlotGen.py

```diff
@@ -235,6 +235,9 @@ def add_plot(self, plot_num, data_frame, title="", test=False):
         fig_params["title"] = title
         fig_params["step_plot"] = self._attr["step_plot"]
         fig_params["fill_graph"] = self._attr["fill"]
+        fig_params["fill_alpha"] = self._attr.get("fill_alpha", 0.3)
```

Comment:
Why 0.3 for the default value?  Isn't 1.0 a better default?  I know that it is not used, but I want to avoid looking at this in three months time and think "there must be a reason for this magic 0.3"